mautic/mautic
# Columns of mapped objects can be added to the form results report

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 


#### Description:


Fields of the object mapped to the form field can be added to the list of form results report columns. E.g. fields of contact, company and custom object.

![image](https://github.com/mautic/mautic/assets/96085911/d9fb0e0e-26c1-429a-bb3f-60ab992ce613)

![image](https://github.com/mautic/mautic/assets/96085911/3821b0eb-37a4-4307-b43b-e3a9b9fb66e9)

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Ensure that the setting `Use the form result as report data sources` is enabled.
3. Create a new form with one or more fields mapped to the object fields (contact, company). 
4. Submit the form.
5. Create a new report using the form result as a data source.
6. Select the data tab. The object fields here.
7. Save the report.
